src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Esfa.Recruit.Provider.Web.ViewModels.Reports.ProviderApplicationsReport;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+
+namespace Esfa.Recruit.Provider.Web.Orchestrators.Reports
+{
+    public class ProviderApplicationsReportOrchestrator
+    {
+        private readonly IRecruitVacancyClient _vacancyClient;
+
+        public ProviderApplicationsReportOrchestrator(IRecruitVacancyClient vacancyClient)
+        {
+            _vacancyClient = vacancyClient;
+        }
+
+        public ProviderApplicationsReportCreateViewModel GetCreateViewModel()
+        {
+            return new ProviderApplicationsReportCreateViewModel { };
```

Comment:
Remove redundant initializer?

Reply:
This will eventually have properties set. So OK for now.